src/React/Basic/Emotion.purs

```diff
@@ -145,3 +160,69 @@ url (URL url') = str ("url(" <> url' <> ")")
 
 color :: Color -> StyleProperty
 color = str <<< cssStringHSLA
+
+-- Absolute length units
+
+-- | Pixels. This function does not take a `Number` because approaches to
+-- | subpixel rendering vary among browser implementations.
+px :: Int -> StyleProperty
+px x = str $ Int.toStringAs Int.decimal x <> "px"
```

Comment:
This is `Number` as well, technically. Perhaps a `px'` for that case would be good since `Int` is more common?

Reply:
Yes, it technically takes a `Number`, but approaches to subpixel rendering vary among browser implementations, so that `13.5px` may be displayed differently in different browsers, so I thought of being a bit more opinionated about that here.

Maybe the flexibility of having a `px' :: Number -> StyleProperty` is worth it?